rust/main/chains/hyperlane-sealevel/src/mailbox.rs

```diff
@@ -128,8 +113,8 @@ impl SealevelMailbox {
     }
 
     /// Get the provider RPC client.
-    pub fn rpc(&self) -> &SealevelRpcClient {
-        self.provider.rpc()
+    pub fn get_provider(&self) -> &SealevelProvider {
```

Comment:
any point of having this function?
if so, should update the comment. If anything I could see this making sense to keep as an RPC getter but don't feel strongly about it being used or not

Reply:
no, I'll remove it :+1:

Reply:
nvm, its still used xd, let me update the comment